extensions/amp-mega-menu/0.1/amp-mega-menu.css

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+header {
+  z-index: 1000 !important;
+  position: relative;
+}
+
+amp-mega-menu {
+  z-index: 1000 !important;
+  white-space: nowrap !important;
```

Comment:
Does `amp-mega-menu` inherit a `position` from somewhere else? If `amp-mega-menu` is `position: static`, I don't think `z-index` applies?

Reply:
yep, it inherits `position: relative` from `ampshared.css`.